docs/en-US/component/collapse.md

```
@@ -107,12 +117,13 @@ collapse/prevent-collapsing
-| Name          | Description                        | Type                     | Default    |
-| ------------- | ---------------------------------- | ------------------------ | ---------- |
-| name          | unique identification of the panel | ^[string] / ^[number]    | —          |
-| title         | title of the panel                 | ^[string]                | ''         |
-| icon ^(2.8.3) | icon of the collapse item          | ^[string] / ^[Component] | ArrowRight |
-| disabled      | disable the collapse item          | ^[boolean]               | false      |
+| Name                  | Description                                                                                                                           | Type                                       | Default    |
+| --------------------- | ------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------ | ---------- |
+| name                  | unique identification of the panel                                                                                                    | ^[string] / ^[number]                      | —          |
+| title                 | title of the panel                                                                                                                    | ^[string]                                  | ''         |
+| icon ^(2.8.3)         | icon of the collapse item                                                                                                             | ^[string] / ^[Component]                   | ArrowRight |
+| collapsible           | Specify whether the panel be collapsible or the trigger area of collapsible (Priority is higher than Collapse Attributes collapsible) | ^[enum]`'icon' \| 'header' \| 'disabled' ` | -          |
```

markdownlint-cli2 (0.18.1)

Spaces inside code span elements

(MD038, no-space-in-code)